README.rst

```diff
@@ -88,28 +88,33 @@ line if you prefer a different location. Example package.json::
 
   {
      "scripts": {
-        "build": "fedx-scripts webpack --config=./config/webpack.config.js",
-        "start:stage": "fedx-scripts webpack-dev-server --config=webpack.dev-stage.config.js",
+        "build": "fedx-scripts webpack --config ./config/webpack.config.js",
+        "start:stage": "fedx-scripts webpack-dev-server --config webpack.dev-stage.config.js",
         ...
      }
   }
 
+Note, specifying a custom config location for babel may cause issues with other
```

Comment:
I noticed this existing limitation while working through stuff. It's addressable by adding some custom config parameters, but I think we can wait until someone actually needs to do this.